terraform/cp4ba/main.tf

```diff
@@ -9,42 +9,38 @@ data "ibm_resource_group" "group" {
   name = var.resource_group
 }
 
-resource "null_resource" "mkdir_kubeconfig_dir" {
-  triggers = { always_run = timestamp() }
-  provisioner "local-exec" {
-    command = "mkdir -p ${var.cluster_config_path}"
-  }
-}
-
 module "create_cluster" {
-  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/main/modules/roks"
+  source = "github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/roks"
 
   enable               = local.enable_cluster
   on_vpc               = var.on_vpc
-  project_name         = var.project_name
+  project_name         = var.roks_project
   environment          = var.environment
   owner                = var.owner
   resource_group       = var.resource_group
   roks_version         = var.platform_version
+  entitlement          = var.entitlement
   flavors              = var.flavors
   workers_count        = var.workers_count
   datacenter           = var.data_center
   force_delete_storage = true
   private_vlan_number  = var.private_vlan_number
   public_vlan_number   = var.public_vlan_number
+  vpc_zone_names       = var.vpc_zone_names
```

Comment:
I think this can be removed since VPC isn't an option here.

Reply:
Done